app/models/searchgov_domain.rb

```diff
@@ -22,8 +26,8 @@ class DomainError < StandardError; end
   attr_readonly :domain
   attr_reader :response
 
-  scope :ok, -> { where(status: OK_STATUS) }
-  scope :not_ok, -> { where.not(status: OK_STATUS).or(where(status: nil)) }
+  scope :ok, -> { where(status: GOOD_STATUS) }
+  scope :not_ok, -> { where.not(status: GOOD_STATUS).or(where(status: nil)) }
```

Comment:
I am not in love with the naming of these scopes. This is not a blocker but since the scope is referring to the status, maybe something like: 

:status_ok
:status_not_ok

These scopes are a bit more clear to me.